### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by fladi** [kaystrobach/TYPO3.piwik#17]
There have been some bugfixes since the 3.1.0 release. Can we expect a new release sometime soon?

**Comment by kaystrobach** [comment]
@fladi any open bugs you know?

**Comment by kaystrobach** [comment]
yes there are some and the ext should be refactored to namespaces ...

**Comment by a-tze** [comment]
I'd like to see at least a 3.1.1 with the invalid markup fix.
